app/assets/stylesheets/base/_utilities.scss

```diff
@@ -1,3 +1,7 @@
+.hidden {
+  display: none !important;
```

Comment:
It's ok if the answer is "yes" - everyone on the team tends to be skeptical of `!important`(s) tho